internal/models/helmcharts/method_list.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package helmchartorgmodel
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChartListResponse Response from listing Charts.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.organization.fluxcd.helm.repository.chartmetadata.chart.ListChartsResponse
+type VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChartListResponse struct {
+
+	// List of charts.
+	Charts []*VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChart `json:"charts"`
```

Comment:
HelmRepository and chartmetadata are two separate resource types. Please check.

Reply:
these are the models in swagger api
chartmetadat  - `VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChartMetadata`
chart - `VmwareTanzuManageV1alpha1OrganizationFluxcdHelmRepositoryChartmetadataChartChart`
repository - `VmwareTanzuManageV1alpha1ClusterNamespaceFluxcdHelmRepositoryRepository`